### DISCUSSION TITLE
听说github要强制开启两步验证？

### DISCUSSION THREAD

**Discussion by kalxd** [just-talks/tech-talks#60] [1.General]
消息来源：[github 要开启两阶段认证了，有什么好方案吗 ](https://emacs-china.org/t/github/25278)

消息可靠吗？
是不是不开启就不给用了啊？

**Comment by adoyle-h** [comment]
是啊，我记得上个月 github 就发邮件通知了。不开就不能用。
推荐用 authy，它可以导出备份给其他 2FA 客户端用。不推荐 Google Authenticator，它的备份只能它自己用，没法迁移到其他 2FA 客户端。

**Reply by kalxd** [reply]
倒不是如何开启问题。
如果强制验证，我得考虑换平台。

**Reply by adoyle-h** [reply]
为何那么讨厌 2FA？提升账号安全不好吗？

**Reply by kalxd** [reply]
不是讨厌2FA，而是讨厌强制。
npm定期给我发开启2FA邮件，却从不强制。
